agentuity/io/email.py

```
@@ -181,8 +292,88 @@ def html(self) -> str:
     @property
-    def attachments(self) -> list:
+    def attachments(self) -> List["IncomingEmailAttachment"]:
         """
-        Return the attachments of the email.
+        Return the attachments of the email as EmailAttachment objects.
         """
-        return getattr(self._email, "attachments", [])
+        raw_attachments = getattr(self._email, "attachments", [])
+        return [IncomingEmailAttachment(a) for a in raw_attachments]
+
+    async def sendReply(
```

Pylint (3.3.7)

[refactor] Too many arguments (8/5)

(R0913)